core/common/src/MapLayerSettings.ts

```diff
@@ -109,23 +109,23 @@ export class MapSubLayerSettings {
  * @beta
  */
 export interface MapLayerProps {
-  /** Controls visibility of layer */
+  /** Controls visibility of layer. Defaults to 'true' in MapLayerSettings.fromJSON(). */
   visible?: boolean;
-  /** Identifies the map layers source. */
+  /** Identifies the map layers source. If omitted, defaults to 'WMS' in MapLayerSettings.fromJSON().*/
   formatId?: string;
   /** Name */
   name?: string;
   /** URL */
   url?: string;
   /** Source layers. If undefined all layers are displayed. */
   subLayers?: MapSubLayerProps[];
-  /** A transparency value from 0.0 (fully opaque) to 1.0 (fully transparent) to apply to map graphics when drawing, or false to indicate the transparency should not be overridden. Default value: 0. */
+  /** A transparency value from 0.0 (fully opaque) to 1.0 (fully transparent) to apply to map graphics when drawing,
+   * or false to indicate the transparency should not be overridden. Default value: 0.
+   * If omitted, defaults to 0 in MapLayerSettings.fromJSON() */
   transparency?: number;
-  /** True to indicate background is tranparent */
+  /** True to indicate background is transparent.  If omitted, defaults to 'true' in MapLayerSettings.fromJSON() */

```

Comment:
It also defaults to true in MapLayerSource.fromJSON(), no? I would just say it defaults to true.